virt/virtual_machines/virt-using-vtpm-devices.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
+[id="virt-vtpm-devices"]
+= Using virtual Trusted Platform Module devices
+include::_attributes/common-attributes.adoc[]
+:context: virt-using-vtpm-devices
+
+toc::[]
+
+A Virtual Trusted Platform Module (vTPM) device functions similarly to a
+physical Trusted Platform Module (TPM) hardware chip.
+
+You can use a vTPM device with virtual machines (VMs) running any operating
+system. However, Windows 11 normally requires the presence of a physical TPM
+chip to install or boot. A vTPM device allows VMs created from a Windows 11
+image to install or boot without a physical TPM chip.
+
+Additionally, if you do not enable vTPM, then the VM does not recognize a
+TPM device, even if the node has one.
+
+vTPM devices also protect virtual machines (VMs) by temporarily storing secrets
+without physical hardware. However, vTPM discards stored secrets after each VM
+shutdown.
```

Comment:
As I stated in a different comment, I think the intro to the module would be better used here. The current assembly intro is better suited to a concept module (as we discussed). 

A couple of minimalism nits: 
- s/similarly to/like
- remove "Additionally," from line 17
- no need to include `(VMs)` in line 20 because you have it in line 12

I'd also suggest changing the second paragraph to something like this: `You can use a vTPM device with any operating system, but Windows 11 requires the presence of a TPM chip to install or boot. A vTPM device allows VMs created from a Windows 11 image to function without a physical TPM chip.`